styles.css

```diff
@@ -0,0 +1,65 @@
+*{
+  box-sizing:border-box;
+}
+
+
+.u-list-reset {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.u-list-inline li {
+  display: inline-block;
+  padding-left: 10px;
+  padding-left: 10px;
```

Comment:
I would recommend moving these padding rules down to the `.list` ruleset on lines 62- 65